packages/shared/src/components/plus/GiftPlusModal.tsx

```diff
@@ -195,27 +171,24 @@ export function GiftPlusModalComponent({
           </Typography>
           <PlusPlanExtraLabel
             color={PlusLabelColor.Success}
-            label={oneTimePayment.extraLabel}
+            label={giftOneYear?.extraLabel}
             typographyProps={{ color: TypographyColor.StatusSuccess }}
           />
           <Typography type={TypographyType.Body} className="ml-auto mr-1">
-            <strong className="mr-1">{oneTimePayment?.price}</strong>
-            {oneTimePayment?.currencyCode}
+            <strong className="mr-1">{giftOneYear?.price}</strong>
+            {giftOneYear?.currencyCode}
           </Typography>
         </div>
         <Typography type={TypographyType.Callout}>
-          Gift one year of daily.dev Plus for {oneTimePayment?.price}. Once the
+          Gift one year of daily.dev Plus for {giftOneYear?.price}. Once the
           payment is processed, they’ll be notified of your gift. This is a
           one-time purchase, not a recurring subscription.
         </Typography>
-        <Button
-          tag="a"
-          variant={ButtonVariant.Primary}
-          href={`${plusUrl}?giftToUserId=${selected?.id}`}
-          disabled={!selected}
-        >
-          Gift & Pay {oneTimePayment?.price}
-        </Button>
+        <Link href={`/plus/gift/${selected?.id}`} passHref>
+          <Button variant={ButtonVariant.Primary} disabled={!selected} tag="a">
```

Comment:
Did this not have some special UI to show they already plus?

Reply:
That is for when selecting the user to Gift. When the user from the URL has plus already, we redirect automatically.

Reply:
I know, but didn't the design have something specific for these?
It's kind of unclear why I can't select certain people now.

Reply:
Yeah, they are disabled, as shown below with the Plus icon beside their username.

![Screenshot 2025-01-28 at 5 26 29 PM](https://github.com/user-attachments/assets/8a2813d3-ff4d-4103-97d0-3fa3bcba3144)

Reply:
But this is in design, missing that right side.
Otherwise maybe suggestion to add hover tooltip that they already plus to product?
![Screenshot 2025-01-28 at 11 35 45](https://github.com/user-attachments/assets/7b403989-a9f5-4e4f-9992-03f988b361de)

Reply:
Right. Yeah, maybe a tooltip is a better option since there can definitely names that will overflow the right side.